Dockerfile

```diff
@@ -66,7 +65,7 @@ RUN tee -a /etc/pacman.conf <<< '[community-testing]' \
     && tee -a /etc/pacman.conf <<< 'Include = /etc/pacman.d/mirrorlist'
 
 RUN pacman -Syu --noconfirm \
-    && pacman -S sudo git make automake gcc python go autoconf cmake pkgconf alsa-utils fakeroot vim nano --noconfirm \
```

Comment:
@sickcodes this part will need your confirmation;)

Reply:
I didn't see they are used, hope I'm not wrong 😅

Reply:
Everything is perfect except python is required for gibMacOS :)